Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -338,12 +344,9 @@ - (void)searchDisplayController:(UISearchDisplayController *)controller didLoadS
 }
 
 - (void)searchBarCancelButtonClicked:(UISearchBar *)SearchBar {
-
     [self filterContentForSearchText:@"" scope:
-     [[self.searchDisplayController.searchBar scopeButtonTitles] objectAtIndex:[self.searchDisplayController.searchBar
-                                                                                selectedScopeButtonIndex]]];
+     [[self.searchDisplayController.searchBar scopeButtonTitles] objectAtIndex:[self.searchDisplayController.searchBar selectedScopeButtonIndex]]];
     [self.tableView reloadData];
```

Comment:
idk why but these spaces are still here even after i deleted them in my local file :(

Reply:
Next time the pull request should be automatically updated as soon as you push the changes to your branch.